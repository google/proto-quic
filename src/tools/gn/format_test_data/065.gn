source_set("test") {
  some_target_name = ":some_target"
  deps = [ some_target_name, "//last_target", ":another_target" ]
}
