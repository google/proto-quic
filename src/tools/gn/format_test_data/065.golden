source_set("test") {
  some_target_name = ":some_target"
  deps = [
    ":another_target",
    "//last_target",
    some_target_name,
  ]
}
